subsys/bluetooth/host/gatt.c

```diff
@@ -45,6 +45,7 @@
 #include "settings.h"
 #include "gatt_internal.h"
 
+
```

Comment:
This line addition seems unrelated? (and redundant)

Reply:
Yes, I will remove this line.